bin/cron/create_rollup_tables

```diff
@@ -16,7 +17,7 @@ TIME_NOW = DateTime.now.strftime('%F %T').freeze
 #   quotations escaped. Returns nil on nil input.
 def sanitize_string_for_db(unsanitized)
   return nil if unsanitized.nil?
-  unsanitized.gsub(/[\r\n]+/, '').gsub(/"/, '\"')
+  unsanitized.gsub(/[\r\n]+/, '').gsub(/\\|"/, '\\\\\\0')
```

Comment:
I got this lovely substitution from https://apidock.com/ruby/String/gsub#787-Replacing-with-and-match. I'm open to better options here but had trouble coming up with anything.